client/components/RatingsAndReviews/Rating/Factors.jsx

```diff
@@ -53,6 +53,10 @@ const marks = [
 export default function CustomizedSlider({ characteristics }) {
   const classes = useStyles();
 
+  if (characteristics.Quality) {
```

Comment:
Should this be if (!characteristics.Quality) ?

Reply:
Pushed the fix